src/main/java/ca/corefacility/bioinformatics/irida/ria/web/models/tables/TableResponse.java

```diff
@@ -1,20 +1,20 @@
-package ca.corefacility.bioinformatics.irida.ria.web.models;
+package ca.corefacility.bioinformatics.irida.ria.web.models.tables;
 
 import java.util.List;
 
 /**
  * Response sent when items are requested for a table.
  */
 public class TableResponse {
-	private List<TableModel> dataSource;
+	private List<? extends TableModel> dataSource;
 	private Long total;
 
-	public TableResponse(List<TableModel> dataSource, Long total) {
+	public TableResponse(List<? extends TableModel> dataSource, Long total) {
```

Comment:
Should we make this a generic class instead of having the `? extends TableModel` here?  That way when we're using this anywhere in java code we can know what the `TableModel` class is.  That might make it easier for testing too.  Come chat with me if you want more on this.

Reply:
I think I got this figured out.  Please see 0b3345beb955af75451a1fedbf8465ecd553b233 and let me know.